CHANGELOG.md

```diff
@@ -3508,7 +3508,6 @@ Released 2018-09-13
 [`unnecessary_filter_map`]: https://rust-lang.github.io/rust-clippy/master/index.html#unnecessary_filter_map
 [`unnecessary_find_map`]: https://rust-lang.github.io/rust-clippy/master/index.html#unnecessary_find_map
 [`unnecessary_fold`]: https://rust-lang.github.io/rust-clippy/master/index.html#unnecessary_fold
-[`unnecessary_join`]: https://rust-lang.github.io/rust-clippy/master/index.html#unnecessary_join
```

Comment:
@flip1995 not sure why these were removed by `cargo dev update_lints`

Reply:
Me neither. Without registering the lint properly it won't do anything. So try re-running it.

Reply:
I did multiple times, it always removes it. Could this be related to the fact that this lint is in the methods folder?

Reply:
I could add the lines manually but they'd just be erased on the next commit